### ISSUE TITLE
Accessibility: High Contrast: Focus is not visible in "command pallete" window in High Contrast Mode.

### ISSUE THREAD

**Issue by Sachin077** [Microsoft/monaco-editor#652]
monaco-editor version: 0.X.Y
Browser: Chrome, Firefox, Edge
OS: Windows 10
Steps or JS usage snippet reproducing the issue: Open command pallete and set high contrast mode. Verify focus on "Context" menu and "Command Pallete" items.

**Comment by alexandrudima** [comment]
@Sachin077

When setting High Contrast, I can see the focus in the command palette is indicated via a border, is that a problem ?

![image](https://user-images.githubusercontent.com/5047891/36788696-31335e20-1c8f-11e8-99e0-dc01d845c6bf.png)

**Comment by bpasero** [comment]
Closing due to no response in any time.
